src/ApiRequestPanel.js

```diff
@@ -165,13 +164,16 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
   }
 
   _renderServerSelector() {
-    const { amf, hideServerSelector, selectedServerType, selectedServerValue } = this;
+    const { amf, hideServerSelector, selectedServerType, selectedServerValue, serversCount, hideCustomServer } = this;
+    const hidden = (serversCount && serversCount < 1) || !!hideServerSelector;
     return html`<api-server-selector
-      .hidden=${hideServerSelector}
+      .hidden=${hidden}
+      .hideCustom=${hideCustomServer}
```

Comment:
Didn't see this at the time as well. This is terrible name. It should be something like `noCustom` (which is again a boolean attribute so please set it with `?` and not `.`).
Actually the web way to do this, because it is boolean attribute, is to have `custom` attribute that allows to render custom value when the attribute is set. This is better than any `no*` or `hide*` attribute.